internal/utils/helpers.go

```diff
@@ -155,3 +155,19 @@ func HasSuffix(str string, extensions []string) bool {
 func HasJsonOrYamlExt(str string) bool {
 	return HasSuffix(str, []string{".json", ".yaml", ".yml"})
 }
+
+func ansiColorText(str string, color int) string {
```

Comment:
We use https://pkg.go.dev/github.com/doiit/picocolors (that no longer seems to exist) in the acceptance tests. We might want to think about disabling colors based on the environment, picocolors does something like this:

```
func isColorSupported() bool {
        noColor := os.Getenv("NO_COLOR") != "" || contains(os.Args, "--no-color")
        forceColor := os.Getenv("FORCE_COLOR") != "" || os.Getenv("CLICOLOR_FORCE") != "" || contains(os.Args, "--color")
        terminal := isatty.IsTerminal(os.Stdout.Fd()) || isatty.IsCygwinTerminal(os.Stdout.Fd()) || os.Getenv("TERM") == "dumb"
        ci := os.Getenv("CI") != ""

        return !noColor && (forceColor || terminal || ci)
}
```

There are [terminfo](https://pkg.go.dev/search?q=terminfo&m=) packages we could also use to se if the terminal has color support (think CI)

Reply:
Yeah good idea.

Reply:
Third commit has this now.